Utils.py

```diff
@@ -568,6 +569,8 @@ def queuer():
             else:
                 if text:
                     queue.put_nowait(text)
+                else:
+                    sleep(0.01)  # non-blocking stream
```

Comment:
0.001 might be fine? Plenty fast that no human perceives the delay, but checking 1000 times a second is also not a major strain on modern CPUs.

Reply:
I believe 10ms is fine given how kivy typically runs at 60FPS a fully loaded PC will switch tasks slower than10ms and the default windows timer resolution is awful anyway.

If we run into issues with this, I would suggest switching to select() rather than reducing sleep time.